src/map/guild.cpp

```diff
@@ -1262,22 +1262,50 @@ int guild_notice_changed(int guild_id,const char *mes1,const char *mes2) {
 	return 0;
 }
 
+/*====================================================
+ * Check condition for changing guild emblem
+ *---------------------------------------------------*/
+bool guild_check_emblem_change_condition(map_session_data *sd)
+{
+	nullpo_ret(sd);
+	guild* g = sd->guild;
+
+	if (battle_config.require_glory_guild &&
+		g != nullptr && guild_checkskill(g, GD_GLORYGUILD) > 0) {
```

Comment:
```suggestion
	if (battle_config.require_glory_guild && g != nullptr && guild_checkskill(g, GD_GLORYGUILD) > 0) {
```
Just in-line it. We are past our glory days of 4:3 aspect ratio.